test/unit/lib/plugins/aws/package/compile/events/httpApi.test.js

```diff
@@ -204,6 +204,52 @@ describe('lib/plugins/aws/package/compile/events/httpApi.test.js', () => {
       expect(cfStage.Properties.AccessLogSettings).to.have.property('Format');
     });
 
+    it('should not disable default execute-api endpoint', () => {
+      expect(cfApi.Properties.DisableExecuteApiEndpoint).to.equal(undefined);
```

Comment:
Let's use `to.not.have.property`

Reply:
It seems to always have the property even if it is set as `undefined` ?
Excuse my lack of JS knowledge :bow:

Reply:
Ah, indeed, we allow that. In that case let's leave this test as it is. Sorry for confusion